imagine/templates/magnetic_field_template.py

```diff
@@ -15,12 +15,8 @@ class MagneticFieldTemplate(MagneticField):
     STOCHASTIC_FIELD = True
     # If there are any dependencies, they should be included in this list
     DEPENDENCIES_LIST = []
-
-    @property
-    def field_checklist(self):
-        # This property returns a dictionary with all the
-        # available parameters as keys
-        return {'Parameter_A': None, 'Parameter_B': None}
+    # List of all parameters of the field
+    PARAMETERS_LIST = {'Parameter_A': None, 'Parameter_B': None}
```

Comment:
Shouldn't this be a list instead of a dict?

Reply:
Oops, yes it should!